packages/node_modules/@webex/plugin-meetings/sample/app.js

```diff
@@ -60,23 +60,23 @@ function connect() {
       });
   }
 
-  // TODO: handle when other user answers outgoing call, update the HTML
-  spark.meetings.on('meeting:ringing', (m) => {
-    if (m.type === 'INCOMING') {
-      document.getElementById('ringing-call').innerHTML = 'incoming call';
-    }
-    if (m.type === 'OUTGOING') {
-      document.getElementById('ringing-call').innerHTML = 'outgoing call';
-    }
-  });
-
   spark.meetings.on('meeting:added', (m) => {
-    if (m.type === 'INCOMING') {
+    const type = m.type;
+    if (type === 'INCOMING') {
       newMeeting = m.meeting;
       toggleDisplay('incomingsection', true);
+      newMeeting.acknowledge(type);
       newMeeting.on('meeting:alerted', () => {
```

Comment:
What's the purpose of the `alerted` event? I know it was there before this PR but I wasn't aware of it.

Reply:
When we get a incoming toast message we need to notify locus server